plutarch-ledger-api/CHANGELOG.md

```diff
@@ -4,6 +4,22 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
+## WIP
+
+### Added
+
+* `pposixTime` and `unPPosixTime`, to construct and deconstruct `PPosixTime`
+  more conveniently
+
+### Changed
+
+* `PPosixTime` now permits negative values as a result of its operations
+
+### Fixed
+
+* Bug in `PNum` instance for `PPosixTime`, where `#*` was implemented as `#+
```

Comment:
Yikes! TBF this shouldn't even be implemented though. Haskell's way of overloading operators leaves much to be desired.

Reply:
Yes, big yikes.

Reply:
I feel like the better solution would be to drop all the instances completely, make a new prelude where all operators are "untyped", e.g.
```haskell
class OperatorPlus a b c | a b -> c , a c -> b, b c -> a where
  (+) :: a -> b -> c

instance OperatorPlus PPosixTime PInteger PPosixTime where ...
```

Then we wouldn't have all these arguably incorrect instances.